paddle/fluid/platform/float16.h

```diff
@@ -1039,6 +1039,11 @@ HOSTDEVICE inline float16 exp(const float16& a) {
   return float16(::expf(static_cast<float>(a)));
 }
 
+template <>
+HOSTDEVICE inline float16 erf(const float16& a) {
+  return float16(::erff(static_cast<float>(a)));
+}
```

Comment:
Please add unit test for `FP16`.

Reply:
Sorry I missed that by a mistake. It's been added now.